recipes/pymvpa2/meta.yaml

```diff
@@ -0,0 +1,82 @@
+# Use
+# python -c 'from jinja2 import Template; print(Template(open("meta.yaml").read()).render())'
+# to see templating interpolated
+{% set name = "pymvpa2" %}
+{% set version = "2.6.1" %}
+{% set sha256 = "206fa302fc6292831c50d6ffe9b1a95fc048380b0701c140eb7300dcd017052a" %}
+# Common dependencies between run and build (Yarik hates duplication)
+{% set common_deps = "
```

Comment:
I would just get rid of this `common_deps` and list explicitly in the sections below.

Reply:
done... now that it is only `python`, made no sense ;)